src/constants/Constants.js

```diff
@@ -33,7 +33,14 @@ export const LAYOUT = {
   MASKED_INPUT: 'masked-input',
   COLLAPSED: 'collapsed',
   SPARQL: 'sparql',
-  TURTLE: 'ttl'
+  TURTLE: 'ttl',
```

Comment:
Is it a good idea to hardcode a predefined set of categories. And what are these categories even supposed to represent?

Reply:
There are many scenarios for the usage of categories e.g.:
- classify question according to source from which questions were created (e.g. we could have category-1 being questions generated from PSP and category-2 questions generated from Metropolitni plan / similarily in aviation we have questions generated from )
- differentiate personal information and non-personal information
- differentiate questions about ufo:events vs. questions about ufo:objects (in some cases it might be confusing to decide and explicit categories might help)
- based on data collection analysis we can show questions that were not filled in by most of the users (category-1), questions that were entered by most of the users (category-2), questions that were confusing to some users as they entered the value but changed it later (category-3), etc ...

As can be seen, the scenarios indicate that we don't need concrete categories but generic categories that can be used for different purposes. The idea is to provide in the description of the question (i.e. in question-mark) explanation why this category + additional detail, but visualization of the question category helps the user to understand questions without the need to look to the description.

ad hardcoded)
There is also a followup (low priority for me) of this task described in #27, where categories are generated dynamically.
ad what categories supposed to represent?)
Just generic classification.

Reply:
Ok, fair enough.